src/test/java/net/starlark/java/eval/testdata/bench_int.star

```diff
@@ -0,0 +1,25 @@
+_11 = 1 << 11        # 32-bit integer
+_33 = 1 << 33        # 64-bit integer
+_ml = (1 << 63) - 1  # maxlong
+_66 = 1 << 66        # BigInteger
+
+def bench_int_add_32(b):
+    for _ in range(b.n):
+        x = (_11 + _11, _11 + _11, _11 + _11, _11 + _11, _11 + _11,
```

Comment:
I assume the tuple acts an unrolled loop without whatever overheads that has.

The x=... assignment could be omitted.

Reply:
Yes, `(_11 + _11, ...)` 10 times, works about 10% faster than `_11 + _11` invoked in a loop, this way the bench overhead is smaller.